dbms/src/Flash/Executor/PipelineExecutorStatus.h

```diff
@@ -149,8 +164,12 @@ class PipelineExecutorStatus : private boost::noncopyable
     ResultQueuePtr getConsumedResultQueue();
 
 private:
+    const String query_id;
+
     LoggerPtr log;
 
+    MemoryTrackerPtr mem_tracker;
```

Comment:
Looks like that `PipelineExecutorStatus` now holds much more things than `status`, maybe we should rename it to a more proper name like `PipelineExecutorContext`?

Reply:
good catch, all renamed~